modules/openxoutstreamBidAdapter.js

```diff
@@ -0,0 +1,225 @@
+import { config } from 'src/config';
+import { registerBidder } from 'src/adapters/bidderFactory';
+import * as utils from 'src/utils';
+import { BANNER } from 'src/mediaTypes';
+import { parse } from 'src/url';
+
+const SUPPORTED_AD_TYPES = [BANNER];
+const BIDDER_CODE = 'openxoutstream';
+const BIDDER_CONFIG = 'hb_pb_ym';
+const BIDDER_VERSION = '1.0.0';
+const CURRENCY = 'USD';
+const NET_REVENUE = true;
+const TIME_TO_LIVE = 300;
+const YM_SCRIPT = `!function(e,t){if(void 0===t._ym){var a=Math.round(5*Math.random()/3)+'';t._ym='';var m=e.createElement('script');m.type='text/javascript',m.async=!0,m.src='//static.yieldmo.com/ym.'+a+'.js',(e.getElementsByTagName('head')[0]||e.getElementsByTagName('body')[0]).appendChild(m)}else t._ym instanceof String||void 0===t._ym.chkPls||t._ym.chkPls()}(document,window);`;
+const PLACEMENT_ID = '1986307928000988495';
+const LICENSED_FORMAT_ID = '1991358644725162817';
+const PUBLISHER_ID = '1986307525700126029';
+
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: SUPPORTED_AD_TYPES,
+  isBidRequestValid: function(bidRequest) {
+    if (utils.deepAccess(bidRequest, 'mediaTypes.banner') && bidRequest.params.delDomain) {
+      console.log('valid bid request');
+      return !!bidRequest.params.unit || utils.deepAccess(bidRequest, 'mediaTypes.banner.sizes.length') > 0;
+    }
+    return !!(bidRequest.params.unit && bidRequest.params.delDomain);
+  },
+  buildRequests: function(bidRequests, bidderRequest) {
+    if (bidRequests.length === 0) {
+      return [];
+    }
+    let requests = [];
+    requests.push(buildOXBannerRequest(bidRequests, bidderRequest));
+    return requests;
+  },
+  interpretResponse: function(serverResponse, serverRequest) {
+    return handleVastResponse(serverResponse, serverRequest.payload)
+  },
+
+  transformBidParams: function(params, isOpenRtb) {
+    return utils.convertTypes({
+      'unit': 'string',
+    }, params);
+  }
+};
+
+function getViewportDimensions(isIfr) {
+  let width;
+  let height;
+  let tWin = window;
+  let tDoc = document;
+  let docEl = tDoc.documentElement;
+  let body;
+
+  if (isIfr) {
+    try {
+      tWin = window.top;
+      tDoc = window.top.document;
+    } catch (e) {
+      return;
+    }
+    docEl = tDoc.documentElement;
+    body = tDoc.body;
+
+    width = tWin.innerWidth || docEl.clientWidth || body.clientWidth;
+    height = tWin.innerHeight || docEl.clientHeight || body.clientHeight;
+  } else {
+    docEl = tDoc.documentElement;
```

Comment:
this assignment is the same in both branches

Reply:
ya their LGTM tool caught that too. updated.